src/services/profiles/profiles.ts

```diff
@@ -32,11 +43,28 @@ export default (
   },
 
   async create(requestBody, options) {
-    return (await client.post(httpAuth, '/', requestBody, options)).data;
+    const customKeys = ['custom_fields', 'groups.custom_fields'];
```

Comment:
@wwaaijer-exh i have added these based on the swagger file but I think this is not needed, since they can not be added to the profile in a creation, they can probably also not be returned?

Reply:
You're right, it should never be returned. But this seems like the best future proof and consistent thing to do 👌